Source/GRChomboCore/BoundaryConditions.cpp

```diff
@@ -242,9 +363,9 @@ void BoundaryConditions::write_boundary_conditions(const params_t &a_params)
 int BoundaryConditions::get_var_parity(int a_comp, int a_dir,
```

Comment:
In AMRInterpolator.impl.cpp this is called:
"BoundaryConditions::get_vars_parity(comp, dir, ... )" which causes an error on compiling. At the moment I just changed the name in AMRInterpolator.impl.cpp.

Reply:
I don't see this on this branch? Are you sure you had the correct branch checked out?

Reply:
This was a change in this in the previous PR by Tiago I think. The one in this branch should be correct now.